### ISSUE TITLE
Manage alerts better on small links

### ISSUE THREAD

**Issue by mgifford** [microsoft/accessibility-insights-web#580]
## Describe the bug

There are times that the icon for the accessibility problem overlaps the problem. 

You should always be able to see the problem, so that you can understand it without disabling the accessibility insights automated checker.

**To Reproduce**

1. Go to https://tastybackend.com/
2. Enable Automated checks
3. Look at the log in link.

## Expected behavior

The WAVE Toolbar does this correctly.

<img width="552" alt="Screen Shot 2019-04-22 at 3 15 50 PM" src="https://user-images.githubusercontent.com/116832/56520063-8a42e500-6511-11e9-8111-6524b6bd8f81.png">

## Screenshots

<img width="551" alt="Screen Shot 2019-04-22 at 3 11 55 PM" src="https://user-images.githubusercontent.com/116832/56520078-9169f300-6511-11e9-9600-5c12b4de78f8.png">
<img width="564" alt="Screen Shot 2019-04-22 at 3 11 46 PM" src="https://user-images.githubusercontent.com/116832/56520079-92028980-6511-11e9-9837-106798d83dc7.png">

## Extension (please complete the following information)

- OS: MacOS
- Chrome Version: 73.0.3683.103

**Comment by adarshrema** [comment]
Hi @mgifford, thank you for your interest in Accessibility Insights and this feature request. We discussed this as a team and our findings are,

Accessibility Insights for Web chooses not to change how the page under test is rendered - the alert floats over the failure location, rather than being inserted next to the failure location. This is different from how WAVE inserts their alert in the flow of the document, potentially shifting how the page is rendered. We have made a conscious implementation decision regarding this.

The workaround is to toggle the visibility of the alert to see what is under it. All alerts can be hidden/shown by unchecking the checkbox in the automated checks list header (below the “N failures were detected” text), or individual alerts can hidden/shown by unchecking the checkbox in the specific row for the failure.

We are interested in finding out what you think about this and welcome other suggestions or ideas you might have.

**Comment by mgifford** [comment]
I think this is a sufficient explanation and I am closing the issue for now. Thanks!
